definitions/ext-flex_host/definition.yml

```diff
@@ -0,0 +1,30 @@
+domain: EXT
+type: FLEX_HOST
+synthesis:
+  name: displayName
+  identifier: displayName
+  conditions:
+  - attribute: eventType
+    value: FlexHostSample
+  tags:
+    arch:
+      multiValue: false
+    coreCount:
+      multiValue: false
+    displayName:
```

Comment:
this doesn't need to be a tag. Is already the name of the entity, and you can filter by name if required.

Reply:
removed thanks